### ISSUE TITLE
Training doesn't start when using batch_normalization,

### ISSUE THREAD

**Issue by shahinkl** [tensorflow/tensorflow#12635]
I am following the article.
https://www.tensorflow.org/api_docs/python/tf/layers/batch_normalization

When I use the below code, the training doesn't start.

`update_ops = tf.get_collection(tf.GraphKeys.UPDATE_OPS)
  with tf.control_dependencies(update_ops):
    train_op = optimizer.minimize(loss)`

Any help will be greatly appreciated.

**Comment by scotthuang1989** [comment]
what do you mean by saying "don't start"? does the training start with an error or just hang?

**Comment by jart** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
